tests/Integration/Behaviour/Features/Context/Domain/AttributeGroupFeatureContext.php

```diff
@@ -70,6 +70,58 @@ public function assertNamedAttributeGroupExists(string $attributeGroupReference,
         $this->getSharedStorage()->set($attributeGroupReference, $foundGroupId);
     }
 
+    /**
+     * @Given I associate attribute group ":attributeGroupReference" with shops ":shopReferences"
+     *
+     * @param string $attributeGroupReference
+     * @param string $shopReferences
+     *
+     * @return void
+     */
+    public function associateAttributeGroupWithShops(string $attributeGroupReference, string $shopReferences): void
```

Comment:
Same suggestion as the previous context, these two methods should be placed in the upper namespace